cylc/flow/scheduler_cli.py

```diff
@@ -351,8 +351,7 @@ async def _run(parser, options, reg, scheduler):
         ret = 1
     except (KeyboardInterrupt, asyncio.CancelledError):
         ret = 2
-    except Exception as exc:
-        LOG.exception(exc)
+    except Exception:
```

Comment:
IMO we probably need to have the logging of this particular exception, but perhaps it should be at debug level.